app/views/admin/registrations/index.html.haml

```diff
@@ -11,6 +11,26 @@
             = link_to 'Export XLS', {format: :xlsx}, class: 'btn btn-success'
       %p.text-muted
         All the people who registered to your event
+.row
+  .col-sm-4.col-xs-4
+    .dashbox.text-center
```

Comment:
But I think it has no sense to show the registration with a `doughnut_chart`. Maybe we can remove the number of registrations from the statistics, as it is in the title.